src/personnummer.js

```diff
@@ -148,6 +144,41 @@ module.exports = {
     return Math.floor((Date.now() - new Date(parts.century + parts.year, parts.month, day).getTime()) / 3.15576e+10);
   },
 
+  /**
+   * Check if a Swedish social security number is for a female.
+   *
+   * @param {string|number} ssn
```

Comment:
I would love to see param descriptors, while It's not at all urgent and it's something I can do later if I really feel the need! :)

Reply:
;)